src/borg_import/helpers/testsuite/test_timestamps.py

```diff
@@ -18,7 +18,14 @@ def test_datetime_from_mtime(tmpdir):
 
 
 def test_datetime_from_string():
-    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59)
-    assert datetime_from_string('Mon Oct 31 23:35:50 UTC 2016') == datetime(2016, 10, 31, 23, 35, 50)
+    # Start with local timezone, convert to UTC, and then convert back to local.
+    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59).astimezone()
+    # FIXME: strptime discards timezone info, and creates a naive time.
+    # UTC is handled specially, both in strptime and datetime_from_string;
+    # local conversions using this format may or may not work.
```

Comment:
there is no strptime here, so maybe the fixme should be moved elsewhere?

Reply:
Comment has been modified to be more explanatory.